packages/use-presence/src/usePresenceSwitch.tsx

```diff
@@ -1,35 +1,36 @@
 import {useState, useEffect} from 'react';
-import usePresence from './use-presence';
+import usePresence from './usePresence';
 
 export default function usePresenceSwitch<ItemType>(
-  item: ItemType,
+  item: ItemType | undefined,
   opts: Parameters<typeof usePresence>[1]
 ) {
   const [mountedItem, setMountedItem] = useState(item);
-  const [shouldBeMounted, setShouldBeMounted] = useState(!isNil(item));
-  const {isMounted, ...otherStates} = usePresence(shouldBeMounted, opts);
+  const [shouldBeMounted, setShouldBeMounted] = useState(!isUndefined(item));
+  const {isMounted, ...rest} = usePresence(shouldBeMounted, opts);
+
   useEffect(() => {
     if (mountedItem !== item) {
       if (isMounted) {
         setShouldBeMounted(false);
-      } else if (!isNil(item)) {
+      } else if (!isUndefined(item)) {
         setMountedItem(item);
         setShouldBeMounted(true);
       }
-    } else if (isNil(item)) {
+    } else if (isUndefined(item)) {
       setShouldBeMounted(false);
-    } else if (!isNil(item)) {
+    } else if (!isUndefined(item)) {
       setShouldBeMounted(true);
     }
   }, [item, mountedItem, shouldBeMounted, isMounted]);
 
   return {
-    ...otherStates,
-    isMounted: isMounted && !isNil(mountedItem),
+    ...rest,
+    isMounted: isMounted && !isUndefined(mountedItem),
     mountedItem
   };
 }
 
-function isNil<ItemType>(value: ItemType) {
-  return value == null;
+function isUndefined<ItemType>(value: ItemType) {
```

Comment:
So I'm fine with this, but at this point, I feel like I lean towards just using `item === undefined` or `item !== undefined` in places rather than having the extra function 😅

Reply:
Sounds good!